apps/fluent-tester/src/RNTester/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -94,7 +135,7 @@ export const CalloutTest: React.FunctionComponent<{}> = () => {
 
       <View ref={custBtnRef} style={{ flexDirection: 'row' }}>
         <Button content="Press for Callout" onClick={toggleShowCustomizedCallout} />
-        <Text>
+        <Text selectable={true}>
```

Comment:
any reason for this? Guessing some possibly related testing?

Reply:
ahh no reason for this here, was trying to test behavior for another issue -- thanks!